### DISCUSSION TITLE
Shutdown notebooks without any bonds

### DISCUSSION THREAD

**Discussion by ctrekker** [JuliaPluto/PlutoSliderServer.jl#114] [Ideas]
I am exporting notebooks with sliders enabled, but many (not all) of my notebooks exported with PlutoSliderServer have no sliders. Currently if sliders are enabled all notebook sessions are kept running, __even if they have no bonds__. To save on memory, could we shut these processes down? I've written [some code](https://github.com/JuliaPluto/PlutoSliderServer.jl/compare/main...cb/shutdown-noninteractive) that does this, but before submitting a PR I wanted to make sure that this is actually a behavior that makes sense. @fonsp

**Comment by ctrekker** [comment]
PS: in my case shutting down 10 non-interactive notebooks saved about 5 GB of memory!

**Comment by fonsp** [comment]
sounds good! i thought we already had this :)
